rest/data/host_create.go

```diff
@@ -194,7 +193,14 @@ func makeDockerIntentHost(taskID, userID string, createHost apimodels.CreateHost
 		SkipImageBuild:   true,
 	}
 
-	return cloud.NewIntent(d, d.GenerateName(), d.Provider, *options), nil
+	hostIntents, err := cloud.GenerateContainerHostIntents(d, 1, *options)
```

Comment:
This isn't the behavior we want. If a task requests a container from this route and all parents are full, we need to insert a new parent intent host to accommodate this, even if that would put is above the maximum that's specified. The max is a cap placed on the scheduler mostly to protect us from it going haywire and spending a lot of money

Reply:
Okay I think the cleanest way to handle this is a boolean set for scheduler and not for host_create so that we can keep this safety check in place. If you have another suggestion for the boolean name lmk, I struggled